src/pages/index.js

```diff
@@ -257,361 +291,348 @@ const Page = () => {
                 Use the Tech Preview
               </BannerCardLink>
             </BannerSubCard>
+
             <BannerSubCard
-              iconName={iconNames.TOOLBOX}
-              headingText="Platforms and Tools"
-              to="/edb-postgres-ai/tools"
+              iconName={iconNames.MIGRATION}
+              headingText="Migration and ETL"
+              to="/edb-postgres-ai/migration-etl"
             >
-              <BannerCardLink to="/edb-postgres-ai/tools/migration-and-ai/">
-                Migration and AI
+              <BannerCardLink to="/edb-postgres-ai/migration-etl/data-migration-service/">
+                Data Migration Service
               </BannerCardLink>
-              <BannerCardLink to="/edb-postgres-ai/tools/management">
-                Management
-              </BannerCardLink>
-              <BannerCardLink to="/edb-postgres-ai/tools/backup">
-                Backup and Recovery
+              <BannerCardLink to="/migration_portal/latest">
+                Migration Portal with AI Copilot
               </BannerCardLink>
+              <BannerCardLink to="/eprs">Replication Server</BannerCardLink>
             </BannerSubCard>
-          </BannerCard>
 
-          <BannerWideCard>
-            <BannerWideCardLink
-              className="col-md-12"
-              to="/repos/"
-              iconName={iconNames.Download}
+            <BannerWideCard>
+              <BannerWideCardLink
+                className="col-md-6"
+                to="/repos/"
+                iconName={iconNames.DOWNLOAD}
+              >
+                Downloads and Repositories
+              </BannerWideCardLink>
+
+              <BannerWideCardLink
+                className="col-md-6"
+                to="/dev-guides/"
+                iconName={iconNames.CODE_WRITING}
+              >
+                Developer Guides
+              </BannerWideCardLink>
+            </BannerWideCard>
+
+            <BannerWideSubCard
+              iconName={iconNames.TOOLBOX}
+              headingText="Platforms and Tools"
+              to="/edb-postgres-ai/tools"
```

Comment:
This heading is clickable, but I get an error. Maybe this should go to: 
```suggestion
              to="/edb-postgres-ai/platforms-and-tools"
```